build/variables.yml

```diff
@@ -69,9 +73,13 @@
   # Flutter version.
   FlutterVersion: '3.22.1'
 
+  # Docker version.
+  DockerVersion: '25.0.5'
+
   # Virtual machine images.
   windowsHostedAgentImage: 'windows-2022'
   macOSHostedAgentImage: 'macOS-13'
+  ubuntuHostedAgentImage: 'ubuntu-latest'
```

Comment:
I would specify the version of ubuntu instead of latest

Reply:
fixed